file/builder.go

```diff
@@ -766,11 +784,47 @@ func (b *stateBuilder) ingestRoute(r FRoute) error {
 	var plugins []FPlugin
 	for _, p := range r.Plugins {
 		p.Route = &kong.Route{ID: kong.String(*r.ID)}
+		if err := b.ingestPluginDefaults(p); err != nil {
+			return err
+		}
```

Comment:
It seems like all of the ingestPluginDefaults are followed by ingestPlugins.

Suggest an alternate concise code path:
- get rid off all existing ingentPluginDefaults() invocations
- add a single one inside ingestPlugins()

Reply:
done